Update the changelog for Jetpack 13.4 (#37269)
Automattic/jetpack: projects/plugins/jetpack/CHANGELOG.md

## 13.4-beta - 2024-05-06
### Enhancements
- Jetpack AI: Add an AI Featured Image Generator. [#36709] [#36806] [#36812] [#36822] [#36831] [#36863] [#36865] [#36868] [#36899] [#36926] [#36963] [#36965] [#37086] [#37166] [#37209] [#37219]
- Newsletters: Manage subscribers in Jetpack Cloud. [#37155]
- Newsletters: Add a "Reply-to" setting for newsletter emails. [#37011]

> - Newsletters: Add filters for controlling the Subscribe Modal load time and the edit capability for the Newsletter Block Neta. [#36909] [#36374]

Reviewer: I suspect this should be "Meta"?

```suggestion
- Newsletters: Add filters for controlling the Subscribe Modal load time and the edit capability for the Newsletter Block Meta. [#36909] [#36374]
```

Author: Yes 😅

> - Paid Content Block: Login flow is matching the Subscribe Block. [#37178]

Reviewer 2: ```suggestion
- Paid Content Block: Use a login flow matching the Subscribe Block. [#37178]
```

- WordAds: Ensure that ads.txt works on subdirectory websites. [#36887]

### Improved compatibility
- General: WordPress 6.4 is now the minimum required version. [#37047] [#37049]

> - Theme Tools: Site-logo and devicepx are deprecated. Use WP Custom Logo instead. [#37116] [#37109]

Reviewer 2: ```suggestion
- Theme Tools: Site-logo and devicepx are deprecated. Use WordPress Core's custom-logo or logo block instead. [#37116] [#37109]
```

- WordPress.com Toolbar: Avoid fatal errors when using other menu management plugins while the WordPress.com Toolbar feature is active. [#36783]

### Bug fixes
- Google Fonts: Avoid theme fonts overriding the default fonts so fonts with a specific font weight that are not supported by the provided theme can be rendered correctly. [#37050]
- Secure Sign-On: Disable the WordPress.com invitation setup on Multisite. [#36690]
- Sidebar: Fix double arrows being shown on active menu items in RTL mode. [#37204]
- Theme Tools: Fix deprecation notices in the color management library. [#36789]
- VideoPress: Fix an XSS vulnerability in VideoPress shortcodes.

### Other changes <!-- Non-user-facing changes go here. This section will not be copied to readme.txt. -->
- AI Assistant: Add acceptance for title optimization. [#37078]
- AI Assistant: Add cancel action for title optimization. [#37080]
- AI Assistant: Add error handling on Title Optimization. [#37195]
- AI Assistant: Add inline extensions feature flag. [#37082]
- AI Assistant: Add partial heading inline extension. [#37087]
- AI Assistant: Add title optimization as beta [#37001]

> - AI Assistant: Add title optimization jetpack sidebar entry. [#37002]

Reviewer: We probably don't care about most of these entries, as they're not user-facing. To be honest, I'm not even sure what most of them mean. But probably should capitalize "Jetpack"?

```suggestion
- AI Assistant: Add title optimization Jetpack sidebar entry. [#37002]
```

- AI Featured Image: Add request counter to UI. [#36744]
- API Docs: Fix example request for media add and update endpoints. [#37072]
- Block Editor: Avoid relying on the Jetpack class. [#37074]
- Block Editor: Fix auto scroll on iframe editor. [#36726]
- Block Editor: Update button props to match latest changes in WordPress 6.5. [#36318]
- Block Editor: Update blocks to use API version 3. [#36701] [#36742] [#36827] [#36852] [#36854] [#36858] [#36859] [#36864]
- Block Scaffolding: Update API version in template. [#36955]
- Blocks: Move localizable strings out of the metadata attributes field. [#36724]
- Business Hours Block: Refactor Edit component and fix time formatting in preview mode. [#36785] [#36786]
- CLI: Make `wp jetpack publicize` work again. [#36988]
- Carousel: Add extra isset check to avoid warnings. [#36681]
- Carousel: Ensure that disabling the feature on WordPress.com Simple brings back Core's Lightbox back. [#36914]
- CloudFlare Analytics: deprecate feature and move to mu-wpcom. [#37061]
- Contact Form: Removing code and renaming relevant references to ensure we use the package version of the contact form. [#37157]
- Cookie Consent: Localize default text. [#37012]

> - Donations Block: Fix non localized strings. [#36951]

Reviewer: ```suggestion
- Donations Block: Fix non-localized strings. [#36951]
```

- Donations Block: Show Stripe nudge if not connected. [#36841]
- Enhanced Distribution: Remove deprecated module file. [#37032]
- Eventbrite Block: Refactor Edit component to function. [#36739]

> - Featured Image: Add 'Carrousel' in 'Featured Image'. [#36834]

Reviewer: ```suggestion
- Featured Image: Add 'Carousel' to 'Featured Image'. [#36834]
```
